yarn.lock

```diff
@@ -3293,6 +3293,11 @@ jju@~1.4.0:
   resolved "https://registry.yarnpkg.com/jju/-/jju-1.4.0.tgz#a3abe2718af241a2b2904f84a625970f389ae32a"
   integrity sha512-8wb9Yw966OSxApiCt0K3yNJL8pnNeIv+OEq2YMidz4FKP6nonSRoOXc80iXY4JaN2FC11B9qsNmDsm+ZOfMROA==
 
+js-base64@^3.7.7:
```

Comment:
The version bump is also sucking in the change that fixes `Property 'atob' doesn't exist` error in React Native 0.72.x for ([PR](https://github.com/Eppo-exp/js-client-sdk-common/pull/75))